README.md

```diff
@@ -99,6 +99,53 @@ const result = await rawResult.json()
 */
 ```
 
+### Get an Account's Public Key
+
+Use Keplr or MetaMask to retrieve an account's public key if it is not returned in the query response.
```

Comment:
would be nice to have more detail on how public keys are required in cosmos transaction since ppl from the evm side might not know.